app/src/main/java/edu/wpi/first/shuffleboard/app/components/Tile.java

```diff
@@ -34,6 +35,8 @@
   private final Property<TileSize> size = new SimpleObjectProperty<>(this, "size", null);
   private final BooleanProperty selected = new PseudoClassProperty(this, "selected");
 
+  private final PropertyBinding<String> titleProperty;
```

Comment:
fixes title bindings getting GC'd

Reply:
Already fixed in #206

Reply:
I thought I remember seeing this.... lol, I thought I pulled before making this PR, must have forgotten.